Parse-Dashboard/Authentication.js

```diff
@@ -81,6 +81,7 @@ function initialize(app, options) {
 function authenticate(userToTest, usernameOnly) {
   var appsUserHasAccessTo = null;
   var matchingUsername = null;
+  var isReadOnly = false;
```

Comment:
switch all these to `let`?  your call :)....

Reply:
yep good call